### ISSUE TITLE
Cannot start ledger after update

### ISSUE THREAD

**Issue by michnovka** [LedgerHQ/ledger-live-desktop#2677]
[ledgerlive-logs-2020.03.05-10.34.13-unversioned.json.zip](https://github.com/LedgerHQ/ledger-live-desktop/files/4292108/ledgerlive-logs-2020.03.05-10.34.13-unversioned.json.zip)

#### Ledger Live Version and Operating System

- tested on Ledger Live **2.0.0**
- Platform and version: **Ubuntu 16.04 LTS 64-bit**

#### Expected behavior
Ledger starts and asks for password.

#### Actual behavior
Ledger shows "Oops, something went wrong" and the following logs:

```
Internal process error (1)
Error
    at createCustomErrorClass (file:///tmp/.mount_ledgerZjsRGB/resources/app.asar/.webpack/renderer.bundle.js:9690:17)
    at deserializeError (file:///tmp/.mount_ledgerZjsRGB/resources/app.asar/.webpack/renderer.bundle.js:9723:25)
    at EventEmitter.handleCommandEvent (file:///tmp/.mount_ledgerZjsRGB/resources/app.asar/.webpack/renderer.bundle.js:257897:61)
    at EventEmitter.emit (events.js:210:5)
    at Object.onMessage (electron/js2c/renderer_init.js:2181:16)
redux failed: Cannot read property 'store' of null
TypeError: Cannot read property 'store' of null
    at ConnectFunction (file:///tmp/.mount_ledgerZjsRGB/resources/app.asar/.webpack/renderer.bundle.js:560:70)
    at Ih (file:///tmp/.mount_ledgerZjsRGB/resources/app.asar/.webpack/renderer.bundle.js:157129:98)
    at si (file:///tmp/.mount_ledgerZjsRGB/resources/app.asar/.webpack/renderer.bundle.js:157149:273)
    at ri (file:///tmp/.mount_ledgerZjsRGB/resources/app.asar/.webpack/renderer.bundle.js:157149:103)
    at pi (file:///tmp/.mount_ledgerZjsRGB/resources/app.asar/.webpack/renderer.bundle.js:157148:178)
    at gk (file:///tmp/.mount_ledgerZjsRGB/resources/app.asar/.webpack/renderer.bundle.js:157241:63)
    at fk (file:///tmp/.mount_ledgerZjsRGB/resources/app.asar/.webpack/renderer.bundle.js:157205:22)
    at Zj (file:///tmp/.mount_ledgerZjsRGB/resources/app.asar/.webpack/renderer.bundle.js:157204:362)
    at Lj (file:///tmp/.mount_ledgerZjsRGB/resources/app.asar/.webpack/renderer.bundle.js:157198:216)
    at file:///tmp/.mount_ledgerZjsRGB/resources/app.asar/.webpack/renderer.bundle.js:157096:115
```
I have an option to restart, when I click on that button, ledger does refresh and shows password prompt. I can log in, but I app does not communicate with ledger device. Its as if it were not connected. It was working fine with previous version. I rebooted PC, changed USB cable, tried back-up ledger device. All same result.

#### Steps to reproduce the behavior
Open ledger 2.0.0

**Comment by F4brtB4425** [comment]
Hey I've been having the same problem since I updated Ledger Live to V2.0 last night. Updated Windows, uninstalled Ledger and deleted the User/AppData/Roaming/Ledger Live folder and reinstalled many times but still getting this ERROR at startup. Can someone please help?

**Comment by F4brtB4425** [comment]
Platform I'm using is Windows 10

**Comment by gre** [comment]
- For Linux, please check this issue: #2670 
- For Windows, please check this issue: https://github.com/LedgerHQ/ledger-live-desktop/issues/2680 

Thanks

**Comment by kushothyleoma** [comment]
I have the exact same problem on Windows 10 64bit. it happened 2 nights ago after I updated to ledger live 2.0.0
I get the oops something went wrong message. so I click restart it loads up I type in my pin. I login but my accounts wont synchronize. I will try my nano ledger s on a different PC and get back

**Comment by F4brtB4425** [comment]
Hey dude, I struggled with this for over 24 hours without sleep, and the only way I was able to solve this problem is to completely uninstall V2.0.0 and go back to the last version which worked flawlessly. In my case it was V1.20.0.


Here is the link on how to do a clean uninstall.

https://www.youtube.com/watch?v=b_QLJSI4VZo



Here is the link to download Ledger Live V1.20.0.

https://github.com/LedgerHQ/ledger-live-desktop/releases/tag/v1.20.0
